api.planx.uk/modules/send/s3/index.ts

```diff
@@ -125,10 +122,11 @@ const sendToS3 = async (req: Request, res: Response, next: NextFunction) => {
         );
       });
 
-    return res.status(200).send({
+    res.status(200).send({
```

Comment:
I guess the `return` here was the original issue - the audit table step wasn't being properly awaited?

Can't quite follow what the exact issue was but it could be caused by mixing `await` and `then()`, which can lead to some issues. It might be worth just using async/await throughout and adding back the `return` here. 

Not sure if this is totally necessary but it might make things a little easier to follow and reason with.

Reply:
Correct! Everything working as expected without this `return` now.

Agree the mix of async/await and an Axios `.then()` are a bit confusing to reason about, but this now exactly matches the pattern we have for auditing BOPS etc (perhaps worth revisiting in whole sometime for consistency!): https://github.com/theopensystemslab/planx-new/blob/main/api.planx.uk/modules/send/bops/bops.ts#L134